src/openrct2/core/StringBuilder.hpp

```diff
@@ -132,6 +133,20 @@ class StringBuilder final
         return result;
     }
 
+    /**
+     * Returns the current string buffer as a standard string.
+     */
+    std::string GetStdString() const
+    {
+        std::string result;
```

Comment:
This should be replaced with the iterator-based constructor:
```
template< class InputIt >
basic_string( InputIt first, InputIt last,
              const Allocator& alloc = Allocator() );
```
so:
`std::string result(_buffer, &_buffer[_length - 1]);`